doc/contributing/docs/style.rst

```diff
@@ -76,4 +70,23 @@ Many people, especially non-native English speakers,
 aren't familiar or don't know the difference between
 `"i.e." and "e.g." contractions
 <https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference>`_.
-So it's best to avoid using them.
\ No newline at end of file
+So it's best to avoid using them.
+
+Specify a link text
+-------------------
+
+While giving a :doc:`link </contributing/docs/markup/links>`, specify clearly
+where this link leads. Thus, you will not mislead a reader.
+
+Bad example:
```

Comment:
👍 

There's a bad example at https://www.tarantool.io/en/doc/latest/contributing/docs/markup/links/#linking-to-external-resources, could you change it too?